API/TreeData/TreeDataMock.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+using System.Collections;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+using MySql.Data;
+using MySql.Data.MySqlClient;
+
+namespace TreeData_CLI
+{
+    public class TreeRequest
+    {
+        public string Subquadrat { get; set; }
+        public string Tag { get; set; }
+        public string StemTag { get; set; }
+        public string SpCode { get; set; }
+        public double DBH { get; set; }
+        public string Codes { get; set; }
+        public string Comments { get; set; }
+        public bool WasDead(ArrayList Trees){
+            foreach(TreeStorage tree in Trees){
+                if(tree.Tag == Tag && tree.StemTag == StemTag
+                    && tree.Codes.Contains("dt") ){
+                    return true;
+                }
+            }
+            return false;
+        }
+    }
+    public class TreeStorage {
+        public int TempID { get; set; }
+        public string QuadratName { get; set; }
+        public string Tag { get; set; }
+        public string StemTag { get; set; }
+        public string SpCode { get; set; }
+        public double DBH { get; set; }
+        public string Codes { get; set; }
+        public string HOM { get; set; }
+        public string ExactDate { get; set; }
+        public double x { get; set; }
+        public double y { get; set; }
+        public string PlotID { get; set; }
+        public string CensusID { get; set; }
+        public string Errors { get; set; }
+
+        public TreeStorage MapSQL(MySqlDataReader rdr){
+            TempID = rdr.GetInt32("TempID");
+            QuadratName = rdr["QuadratName"].ToString();
+            Tag = rdr["Tag"].ToString();
+            StemTag = rdr["StemTag"].ToString();
+            SpCode = rdr["SpCode"].ToString();
+            DBH = rdr.GetDouble("DBH");
+            Codes = rdr["Codes"].ToString();
+            HOM = rdr["HOM"].ToString();
+            ExactDate = rdr["ExactDate"].ToString();
+            x = rdr.GetDouble("x");
+            y = rdr.GetDouble("y");
+            PlotID = rdr["PlotID"].ToString();
+            CensusID = rdr["CensusID"].ToString();
+            Errors = rdr["Errors"].ToString();
+            return this;
+        }
+    }
+    public class TreeResponse
+    {
+        public string Subquadrat { get; set; }
+        public string Tag { get; set; }
+        public string StemTag { get; set; }
+        public int ErrorCode { get; set; }
+        public string Error { get; set; }
+        public TreeResponse (TreeRequest tree, int ecode, string error){
+            Subquadrat = tree.Subquadrat;
+            Tag = tree.Tag;
+            StemTag = tree.StemTag;
+            ErrorCode = ecode;
+            Error = error;
+        }
+    }
+    public static class TreeDataMock

```

Comment:
Rename to `ProcessTrees`

Reply:
Fixed in https://github.com/ForestGeoHack/ForestGEO/pull/12/commits/0543202ea320047257a303f345157882127330c9

Reply:
My apologies, but I meant to fix this comment so that the class name is `TreeTriggers`. It is convention for the class name to match the file name.

Reply:
Renamed in https://github.com/ForestGeoHack/ForestGEO/pull/12/commits/8b20b3ba4a4d8cc5c89f141b429232b8cf71f968